src/components/Hero/ProfileChart/ProfileDetails.js

```diff
@@ -150,7 +174,12 @@ function ProfileDetails({
               <Share
                 title={`${position} ${name} ${promisesByStatus.count} ${promiseLabel} ${trailText}`}
                 link={config.URL}
-              />
+                classes={{
+                  share: classes.share,
+                }}
+              >
+                <SvgIcon component={ShareIcon} viewBox="0 0 24 24" />
```

Comment:
```suggestion
                <SvgIcon component={ShareIcon} viewBox="0 0 20 20" />
```
This needs to match `viewBox` defined in the svg file.